packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -475,4 +476,30 @@ export class MetriportMedicalApi {
 
     return resp.data;
   }
+
+  /**
+   * Returns a signed url to upload a document to Metriport.
```

Comment:
This should indicate how the result should be used to upload a file, form based upload, with the fields as form fields and the url as form url